pkgs/by-name/me/meson/package.nix

```
@@ -18,6 +18,26 @@
 
 let
   inherit (darwin.apple_sdk.frameworks) AppKit Cocoa Foundation LDAP OpenAL OpenGL;
+
+  # Meson relies heavily on Python executable being named python3 for tests, so
+  # we give it a wrapper that executes appropriate executable.
+  pythonWrapper = buildPackages.writeCBin "python3" ''
+    #include <assert.h>
+    #include <errno.h>
+    #include <stdio.h>
+    #include <string.h>
+    #include <unistd.h>
+
+    const char* argv0 = "${python3.executable}";
+
+    int main(int argc, char *argv[]) {
+      assert(argc > 0);
+      argv[0] = argv0;
+      execvp(argv0, argv);
+      fprintf(stderr, "python3: %s: %s\n", argv0, strerror(errno));
+      return 1;
+    }
+  '';
 in
 python3.pkgs.buildPythonApplication rec {
```

We need to verify if it can be replaced by 
```suggestion
python3Packages.buildPythonApplication rec {
```